src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -881,7 +880,7 @@ const WorkflowView = _.flow(
               div({ role: 'radiogroup', 'aria-label': 'Select number of target entities', style: { marginBottom: '1rem' } }, [
                 div([
                   h(RadioButton, {
-                    disabled: !!WorkspaceUtils.editWorkspaceError(ws) || currentSnapRedacted,
+                    disabled: WorkspaceUtils.canEditWorkspace(ws).value || currentSnapRedacted,
```

Comment:
Shouldn't this be disabled is the user can _not_ edit the workspace?

```suggestion
                    disabled: !WorkspaceUtils.canEditWorkspace(ws).value || currentSnapRedacted,
```

Same comment for several other uses in this file.